bm-persona/Common/TagView.swift

```diff
@@ -51,3 +51,19 @@ class TagView: UILabel {
     }
 
 }
+
+// Common TagViews
+extension TagView {
+    
+    static var open: TagView {
+        let color = UIColor(displayP3Red: 133/255, green: 158/255, blue: 248/255, alpha: 1.0)
```

Comment:
Move these colors to Assets/Color.swift?

Reply:
Done!